### PR TITLE
Rel1/hb1

### PR DESCRIPTION
**Pull request by manojgirachh77**
image

### CODE REVIEW

File: src/main/java/com/symtoms/checker/alexa/handler/NoIntentHandler.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		String speechText = "";
+		
+		Object typeObject = getSessionAttributes(input,"type");
+		if(null != typeObject && typeObject instanceof String) {
+			String type = (String) typeObject;
+			switch (type) {
+			case "BodyLocation":
+				return bodyLocationHandler.handle(input);
+			default:
+				speechText = "Sorry, I do not understand how to process that.";
```
Comment (garganshulgarg) [inline src/main/java/com/symtoms/checker/alexa/handler/NoIntentHandler.java:29]: Better to move this validation in VM file

File: src/main/java/com/symtoms/checker/alexa/handler/YesIntentHandler.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		setSessionAttributes(input, "user_option", Boolean.TRUE);
+		if(null != typeObject && typeObject instanceof String) {
+			String type = (String) typeObject;
+			switch (type) {
+			case "help":
+				return helpHandler.handle(input);
+			case "launch":
+				return bodyLocationHandler.handle(input);
+			default:
+				speechText = "Sorry, I do not understand how to process that.";
```
Comment (garganshulgarg) [inline src/main/java/com/symtoms/checker/alexa/handler/YesIntentHandler.java:33]: Better to move this validation in VM file

File: src/main/resources/main-spring-config.xml [hunk 3]
```diff
@@ -70,18 +79,21 @@
...
 		class="com.symtoms.checker.alexa.integration.client.impl.DefaultDiagnosisClient" />
-	
-	<!-- Importing separate Spring Rest Template configurations -->
-	<!-- Importing separate Spring Integration configurations -->
-	<import resource="classpath:resources/integration/symtoms-checker-common-integration.xml"/>	
-	
+	<bean id="symtomsCheckerService"
+		class="com.symtoms.checker.alexa.service.impl.DefaultSymtomsCheckerService" />
+			
+	<bean id="symtomsRestTemplate" class="org.springframework.web.client.RestTemplate" />
```
Comment (garganshulgarg) [inline src/main/resources/main-spring-config.xml:96]: Do we need this Rest Template now !! I think currently we are using SDK Client provided by API Medic.

File: src/main/java/com/symtoms/checker/alexa/handler/YesIntentHandler.java (hunk 2 above)
Comment (manojgirachh77) [reply src/main/java/com/symtoms/checker/alexa/handler/YesIntentHandler.java:33]: Limitation of Framework

File: src/main/java/com/symtoms/checker/alexa/handler/NoIntentHandler.java (hunk 1 above)
Comment (manojgirachh77) [reply src/main/java/com/symtoms/checker/alexa/handler/NoIntentHandler.java:29]: Limitation of Framework

File: src/main/resources/main-spring-config.xml (hunk 3 above)
Comment (manojgirachh77) [reply src/main/resources/main-spring-config.xml:96]: Not for this Release, we might require in next release, if we integrate with practo api

**Pull request merged**